lib/src/client.dart

```diff
@@ -67,7 +67,7 @@ abstract class MeiliSearchClient {
 
   /// Find index by matching [uid] and responds with raw information from API.
   /// Throws error if index is not exists.
-  Future<Map<String, Object?>> getRawIndex(String uid);
```

Comment:
I mean, does this really bring a "benefit"? Since the users will not typecast this value like we have in the paginated results and search results...

Reply:
personally I haven't really used it, but maybe someone would?

also related: https://github.com/dart-lang/sdk/issues/46017#issuecomment-1415991484